### DISCUSSION TITLE
Network.trr.mode

### DISCUSSION THREAD

**Discussion by mockinbirdy** [bromite/bromite#2396] [Q&A]
Hi, any way to change "network.trr.mode" in bromite? Like it can be manually changed in Firefox.

**Comment by peternrdstrm** [comment]
You can enable/disable/customize DNS Over HTTPS in the setting, trr mode is bacially the same

**Reply by mockinbirdy** [reply]
I know that. I want to change 'network.trr.mode' the way I can in firefox, e.g. from 2 to 3. Haven't found this setting anywhere in bromite.

**Reply by peternrdstrm** [reply]
because they use different engines, their settings will have different names, but the behavior is mostly the same.

**Reply by mockinbirdy** [reply]
Well, ok, could you tell me how this exact setting is called in bromite? Cause in firefox there are separate setting pages for DoH and for modifying network.trr.mode.
